aten/src/ATen/core/TensorOptions.h

```diff
@@ -614,4 +614,10 @@ inline DeviceType computeDeviceType(TensorTypeId tid) {
   }
 }
 
+} // namespace at
+
+namespace c10 {
+
+using TensorOptions = at::TensorOptions;
```

Comment:
why adding new things to c10 namespace while we are in at?

Reply:
It's to keep backwards compatibility with users of the c++ API who may be referring to TensorOptions as `c10::TensorOptions`.